packages/react/src/primitives/Tabs/__tests__/Tabs.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react';
+import { Tabs, TabItem } from '../Tabs';
+import { ComponentClassNames } from '../../shared';
+import { ComponentPropsToStylePropsMap } from '../../types';
+import { kebabCase } from 'lodash';
+
+describe('Tabs: ', () => {
```

Comment:
Regretfully, I've yet to write tests for Tabs. Here are some ideas:

Tabs should:
- only accept `TabItem` components as children
- initially display the content panel of a specific tab based on the `defaultTab` prop, which is an index that defaults to zero (the first tab)
- apply different `className`s at the `Tabs` level and the `TabItem` level
- take up remaining space differently depending on whether `equal` or `relative` was passed to the `grow` prop
- apply any `FlexStyleProps`

TabItems should:
- accept a string or custom component (e.g., `<Text/>`) as the `title` prop
- render certain tabs unclickable based on the `isDisabled` prop

Is there a way to test Accessibility? Radix should be taking care of it though.